pnpm-lock.yaml

```diff
@@ -1,934 +1,3653 @@
-lockfileVersion: '6.0'
+lockfileVersion: '9.0'
```

Comment:
Not sure if we should keep this update out of scope or not.

Reply:
Since I am not doing any changes in dependencies, I just copied back your lock file.